README.md

```diff
@@ -168,6 +168,11 @@ docker-compose run --rm web black .
 docker-compose run -p 9001:9001 watch npm run storybook
 ```
 
+### Running tests
```

Comment:
I just updated the handbook to account for projects moving away from `tox` so you can remove this section. I also added a note about using the `black` formatter, so you can also remove that if you want to